src/core/services/auth.service.spec.ts

```diff
@@ -2,7 +2,7 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { UserService } from './user.service';
 import { User } from '../../infrastructure/entities/user.entity';
 import { getRepositoryToken } from '@nestjs/typeorm';
-import { UserDto } from '../dtos/userDto';
+import { UserDto } from '../dtos/user';
```

Comment:
@Saela See the file for the `Tour` DTOs: Because we're most likely going to have more than just one `User` DTO (e.g. for registration and account managament), I felt it best to already prepare it and have it in a uniform way.

Another solution would be to have a file for each DTO, but I've read a bit on that and the consensus seems to lean towards exporting related DTOs from one file.